libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -391,7 +391,10 @@ export abstract class BaseInstallManager {
       subspace.getSubspaceTempFolder(),
       undefined,
       undefined,
-      extraNpmrcLines
```

Comment:
```ts
      // Look for a global .npmrc-global file
      const globalNpmrcPath: string = `${this.rushConfiguration.commonRushConfigFolder}/.npmrc-global`;
```

@william2958 Let's update [rfc-4230-rush-subspaces.md](https://github.com/microsoft/rushstack/blob/main/common/docs/rfcs/rfc-4230-rush-subspaces.md) to document this file (and any other new config files introduced by the Subspaces feature).

Adding a `rush init` template would also be a good form of documentation, although if you think it's likely that the file path/semantics might change in the future, we can procrastinate that.